src/TLuaInterpreter.cpp

```diff
@@ -9953,18 +9935,15 @@ int TLuaInterpreter::echoPopup(lua_State* L)
     }
 
     Host& host = getHostFromLua(L);
-    QString txt = a2.c_str();
-    QString name = a1.c_str();
-    if (_commandList.size() != _hintList.size()) {
-        lua_pushstring(L, "Error: command list size and hint list size do not match cannot create popup");
-        lua_error(L);
-        return 1;
+    if (commandList.size() != hintList.size()) {
+        lua_pushfstring(L, "echoPopup: commands and hints list aren't the same size");
```

Comment:
`are not` instead of `aren't` perhaps?

Also `tooltips` not `hints` to match other error message changes.

Reply:
Should actually be `hints` elsewhere for consistency, adjusted